### PR TITLE
Add labels to produce richer metrics

### PR DESCRIPTION
**Pull request by androa**

### CODE REVIEW

File: src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt [hunk 1]
```diff
@@ -100,7 +101,14 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
     private fun hentInngåendeJournalpost(journalpostId: String): Behov {
         val journalpost = journalpostArkiv.hentInngåendeJournalpost(journalpostId)
-        jpCounter.inc()
+        jpCounter
+            .labels(
+                journalpost.tema,
+                journalpost.dokumentListe.first().navSkjemaId,
```
Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]: This looks ok - but I do remember there is an issue with label names, they cannot have `null` values, they will either be `"null"` or an NPE in runtime. Either way - it is safer to have an  eg. `orElse("unknown")`

File: src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt [hunk 2]
```diff
@@ -34,6 +34,7 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
     private val jpCounter: Counter = Counter.build()
         .name("dagpenger_journalpost_mottatt")
```
Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]:
```suggestion
         .namespace("dagpenger")
        .name("journalpost_mottatt")
```

Comment (geiralund) [reply src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]: we should enforce namespace - maybe a central prometheus class/factory

File: src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt (hunk 1 above)
Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]:
```suggestion
                 journalPost.dokumentListe.firstOrNull()?.navSkjemaId?: "unknown",
```

Comment (androa) [reply src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]: 👍

File: src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt [hunk 3]
```diff
@@ -100,7 +101,14 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
     private fun hentInngåendeJournalpost(journalpostId: String): Behov {
         val journalpost = journalpostArkiv.hentInngåendeJournalpost(journalpostId)
-        jpCounter.inc()
+        jpCounter
+            .labels(
+                journalpost.tema,
```
Comment (geiralund) [inline src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt]: minor - but `tema` will always be `DAG` as we filter out other "temas" in the kafka stream.

**Comment by geiralund** [comment]
Rebased this to current master head

**Pull request merged**